### DISCUSSION TITLE
[question] why I can see "Running pod on a shutdown master"?

### DISCUSSION THREAD

**Discussion by hchyue** [rancher/rke2#5649] [Q&A]
<img width="705" alt="image" src="https://github.com/rancher/rke2/assets/69490488/632ee472-2b80-46cb-bb82-6729f64c2dc2">
all node joined to node37. after  I shutdown node37,  I run command ' kubectl get pod -A | grep node37' from node38, display some pod  still running on node37.  In fact they are not running. confused

<img width="1307" alt="image" src="https://github.com/rancher/rke2/assets/69490488/c897ebc7-e61e-4b40-9ff3-581be6e544ae">

**Comment by brandond** [comment]
Pod status is reported by the Kubelet. When you stop or shut down the node, the kubelet is no longer running to report the status of pods. Kubernetes does not know if they are running or not, so it just leaves them in the last state reported by the node. This is normal behavior. What were you expecting to see?

**Reply by hchyue** [reply]
Thank you very much for your explanation
